tests/integration/integration.go

```diff
@@ -33,20 +35,30 @@ import (
 	_ "github.com/dapr/dapr/tests/integration/suite/sentry"
 )
 
+var focusF = flag.String("focus", ".*", "Focus on specific test cases. Accepts regex.")
```

Comment:
Seems `filter` would be better.

Reply:
I like focus because it is consistent with [GinkGo](https://onsi.github.io/ginkgo/#focused-specs) which is a popular Go test library/program.